torch/csrc/autograd/python_variable.cpp

```diff
@@ -255,14 +255,18 @@ void activateGPUTrace() {
 }
 
 PyObject* THPVariable_Wrap(const at::TensorBase& var) {
+  return THPVariable_Wrap(at::TensorBase(var));
```

Comment:
Does this slice the variable var?

Reply:
Good question. After investigating the code, I think object slice in Tensor->TensorBase is valid. However, the previous fix is still reasonable since distinguishing const& and && helps more optimisations.

Reply:
I want to use perfect forwarding in the file also. OK, let's do it.